pkg/kafsar/group_coordinator_impl.go

```diff
@@ -246,5 +246,6 @@ func (gci *GroupCoordinatorImpl) HandleLeaveGroup(groupId string,
 		consumerMetadata.consumer.Close()
 	}
 	group.consumerMetadata = nil
+	delete(gci.groupManager, groupId)
```

Comment:
should delete group only one consumer leave group?

Reply:
we should split this bug fix to another PR.

Reply:
there delete the group member, if not delete group, when consumer rejion, member id will be empty and it will sync group err